build-farm/platform-specific-configurations/aix.sh

```diff
@@ -47,17 +48,19 @@ fi
 echo LDR_CNTRL=$LDR_CNTRL
 
 BOOT_JDK_VERSION="$((JAVA_FEATURE_VERSION-1))"
+# shellcheck disable=SC2116
 BOOT_JDK_VARIABLE="JDK$(echo $BOOT_JDK_VERSION)_BOOT_DIR"
```

Comment:
We have a few other versions of this where it has been replaced with e.g.
```suggestion
BOOT_JDK_VARIABLE="JDK${BOOT_JDK_VERSION}_BOOT_DIR"
```

Is thre a reason not to do the same here?

Reply:
I think that one slipped through the net. I'll go through and clean up these manually